jena-extras/jena-serviceenhancer/src/main/java/org/apache/jena/sparql/service/enhancer/impl/util/VarScopeUtils.java

```diff
@@ -37,29 +37,6 @@
  */
 public class VarScopeUtils {
 
-    public static Map<Var, Var> reverseVarRenameMap(Collection<Var> vars) {
```

Comment:
Moved these block of methods to the end of the class - probably they can be eventually removed.